project/plugins.sbt

```diff
@@ -25,7 +25,9 @@ addSbtPlugin("com.eed3si9n" % "sbt-unidoc" % "0.3.3")
 
 addSbtPlugin("com.thoughtworks.sbt-api-mappings" % "sbt-api-mappings" % "0.2.2")
 
-addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.2.3")
+addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.2.6")
+
+addSbtPlugin("pl.project13.sbt" % "sbt-jol" % "0.1.1")
```

Comment:
It is never used "in code", however it makes the `jol:...` commands available in the sbt console.